cafy_pytest/cafy_gta.py

```diff
@@ -121,7 +122,15 @@ def patch_set_or_get_methods_for_test_instance(self, item):
                             continue
                         else:
                             original_method = getattr(class_obj, method_name)
-                            setattr(class_obj, method_name, self.measure_time_for_set_or_get_methods(original_method,class_name))
+                            # if method is non-infra and method belong to cafykit/lib/feature_lib or cafykit/lib/hw and ends with base.py,
+                            # then applying decorator for collecting time on  get and set methods of non-infra methods
+                            # else  if method is infra then applying decorator for collecting time on  get and set methods of infra methods
+                            file_path = original_method.__code__.co_filename
+                            if 'lib/feature_lib' in file_path or 'lib/hw' in file_path:
```

Comment:
if we dont include lib/utils ..we might miss infra methods

Reply:
Hi Nishant for infra methods , i am handling in else part: https://github.com/cafykit/cafy-pytest/pull/135/files#diff-03b28462ba86f5aa773f13c5b3cb87804d3d7d97bcc5cbff39a9d8a29756a7fdR132